Release notes 2.14.13 (#11558)
PrefectHQ/prefect: RELEASE-NOTES.md

- Implement `SendNotification` action type for programmatic Automations — https://github.com/PrefectHQ/prefect/pull/11471

### Documentation
- Add documentation for Incidents feature in Prefect Cloud 
    — https://github.com/PrefectHQ/prefect/pull/11504
    - https://github.com/PrefectHQ/prefect/pull/11532
    - https://github.com/PrefectHQ/prefect/pull/11506
    - https://github.com/PrefectHQ/prefect/pull/11508
- Add security readme — https://github.com/PrefectHQ/prefect/pull/11520
- Add conditional pause example to flow documentation — https://github.com/PrefectHQ/prefect/pull/11536
- Add API modules to Python SDK docs — https://github.com/PrefectHQ/prefect/pull/11538
- Update human-in-the-loop documentation — https://github.com/PrefectHQ/prefect/pull/11497
- Formatting fixes for quickstart and tutorial — https://github.com/PrefectHQ/prefect/pull/11502
- Fix typo in quickstart — https://github.com/PrefectHQ/prefect/pull/11498
- Fix broken link — https://github.com/PrefectHQ/prefect/pull/11507
- Fix method name typo in Tasks tutorial — https://github.com/PrefectHQ/prefect/pull/11523
- Remove redundant word typo — https://github.com/PrefectHQ/prefect/pull/11528

> 

Reviewer: ```suggestion
### Collections
- Add `LambdaFunction` block to `prefect-aws` to easily configure and invoke AWS Lambda functions - https://github.com/PrefectHQ/prefect-aws/pull/355
```

See the following PR for implementation details:
- https://github.com/PrefectHQ/prefect/pull/11503

## Introducing conditional task retries for enhanced workflow control
In this release, we're excited to introduce the ability to conditionally retry tasks by passing in an argument to `retry_condition_fn` in your task decorator, enabling more nuanced and flexible retry mechanisms. This adds a significant level of control and efficiency, particularly in handling complex or unpredictable task outcomes. For more information on usage, check out our [docs](https://github.com/PrefectHQ/prefect/pull/11535)!

See the following PR for implementation details:
- https://github.com/PrefectHQ/prefect/pull/11500

### Enhancements
- Add `prefect cloud open` to open current workspace in browser from CLI — https://github.com/PrefectHQ/prefect/pull/11519
- Implement `SendNotification` action type for programmatic Automations — https://github.com/PrefectHQ/prefect/pull/11471
- Display work queue status details via CLI — https://github.com/PrefectHQ/prefect/pull/11545

> 

Reviewer 2: ```suggestion
- Allow users to add date ranges "Around a time" when filtering by date - https://github.com/PrefectHQ/prefect-design/pull/1069

```

### Fixes
- Validate deployment name in `.deploy` — https://github.com/PrefectHQ/prefect/pull/11539
- Ensure `flow.from_source` handles remote git repository updates — https://github.com/PrefectHQ/prefect/pull/11547

### Documentation
- Add documentation for Incidents feature in Prefect Cloud 
    — https://github.com/PrefectHQ/prefect/pull/11504
    - https://github.com/PrefectHQ/prefect/pull/11532
    - https://github.com/PrefectHQ/prefect/pull/11506
    - https://github.com/PrefectHQ/prefect/pull/11508
- Add security README — https://github.com/PrefectHQ/prefect/pull/11520
- Add conditional pause example to flow documentation — https://github.com/PrefectHQ/prefect/pull/11536
- Add API modules to Python SDK docs — https://github.com/PrefectHQ/prefect/pull/11538
- Update human-in-the-loop documentation — https://github.com/PrefectHQ/prefect/pull/11497

> - Formatting fixes for quickstart and tutorial — https://github.com/PrefectHQ/prefect/pull/11502

Author: ```suggestion
- Improve formatting in quickstart and tutorial — https://github.com/PrefectHQ/prefect/pull/11502
```
